src/shadowbox/model/access_key.ts

```diff
@@ -27,12 +27,12 @@ export interface ProxyParams {
   readonly password: string;
 }
 
-// Parameters needed to enforce an access key data quota, over a sliding window.
+// Parameters needed to enforce an access key data transfer quota, over a sliding window.
 export interface AccessKeyQuota {
-  // The allowed metered traffic measured in bytes.
-  readonly quotaBytes: number;
+  // The allowed metered data transfer measured in bytes.
+  readonly quota: {bytes: number};
```

Comment:
Should we call it `data`, `allowedData` or `dataLimit` to avoid the `quota` stuttering?

Having "allowed" or "limit" also makes it super clear it's not the used data.

Reply:
Done in 93cc9c9.